OneSignalSDK/onesignal/core/src/test/java/com/onesignal/core/internal/startup/StartupServiceTests.kt

```diff
@@ -68,35 +82,34 @@ class StartupServiceTests : FunSpec({
         val mockStartupService1 = spyk<IStartableService>()
         val mockStartupService2 = spyk<IStartableService>()
 
-        val startupService = StartupService(listOf(), listOf(mockStartupService1, mockStartupService2))
+        val startupService = StartupService(setupServiceProvider(listOf(), listOf(mockStartupService1, mockStartupService2)))
 
         // When
-        startupService.start()
+        startupService.scheduleStart()
 
         // Then
         verify(exactly = 1) { mockStartupService1.start() }
```

Comment:
This will probably fail most of the time since the work in `scheduleStart()` probably won't finish in time.

Reply:
Hmm for some reason it works on my local environment. I added a short delay before verifying the mocks and it looks like it's working now.